evap/grades/tests.py

```diff
@@ -238,3 +239,30 @@ def test_403_on_archived_semester(self):
             Evaluation, course=baker.make(Course, pk=1, semester=archived_semester), state=Evaluation.State.PREPARED
         )
         self.app.get("/grades/semester/1/course/1", user=self.grade_publisher, status=403)
+
+
+class GradeEditTest(WebTest):
+    def test_grades_headlines(self):
+        # check midterm headline
+        grade_publisher = baker.make(
+            UserProfile,
+            email="grade_publisher@institution.example.com",
+            groups=[Group.objects.get(name="Grade publisher")],
+        )
+        grade_document = baker.make(GradeDocument)
+        response = self.app.get(
+            f"/grades/semester/{grade_document.course.semester.pk}/course/{grade_document.course.pk}/edit/{grade_document.pk}",
```

Comment:
You could save this into a variable `url` once and make the other two uses shorter, your call :)

Reply:
I think this way ist shows what it does because it changes settings in between.